tests/users/test_admin_views.py

```diff
@@ -459,3 +457,37 @@ def test_profile_check_inconsistency_check(admin_client):
             )
         ],
     )
+
+
+group_permissions_markup = (
+    '<select name="groups" id="id_groups" multiple class="selectfilter" data-field-name="groupes" data-is-stacked="0">'
+)
+user_permissions_markup = (
+    '<select name="user_permissions" id="id_user_permissions" multiple class="selectfilter" '
+    'data-field-name="permissions de l’utilisateur" data-is-stacked="0">'
+)
+
+
+@pytest.mark.parametrize("superuser", [False, True])
+def test_change_hides_permission_section_on_regular_users(client, superuser):
```

Comment:
Pour encore moins de logique dans le test :grin:.
```suggestion
@pytest.mark.parametrize("superuser,assertion", [
    (False, assertNotContains),
    (True, assertContains),
])
def test_change_hides_permission_section_on_regular_users(client, superuser, assertion):
```

Reply:
Malin !